.eslint.yaml

```diff
@@ -0,0 +1,13 @@
+parserOptions:
+    ecmaVersion: 6
+env:
+    es6: true
+    jasmine: true
+    node: true
+extends: eslint:recommended
+rules:
+    arrow-body-style:
+        - error
+        - always
+    arrow-parens: error
+    arrow-spacing: error
```

Comment:
Didn't want to take a straight copy of it at first, so I'll update this from the open token one.